chainermn/extensions/multi_node_early_stopping_trigger.py

```diff
@@ -0,0 +1,41 @@
+import random
+import string
+from chainer.training.triggers import EarlyStoppingTrigger
+from chainermn.extensions import ObservationAggregator
+
+
+def _random_suffix(n):
+    return ''.join([random.choice(string.ascii_letters) for i in range(n)])
+
+
+class MultiNodeEarlyStoppingTrigger(object):
+
+    def __init__(self, comm, check_trigger=(1, 'epoch'), monitor='main/loss',
+                 patience=None, mode='auto', verbose=False,
+                 max_trigger=(100, 'epoch'), **kwargs):
+
```

Comment:
Could you add the description about `MultiNodeEarlyStoppingTrigger` ?
ref:
https://docs.chainer.org/en/stable/reference/generated/chainer.training.triggers.EarlyStoppingTrigger.html

Reply:
fixed.